dnssd/src/main/java/com/github/druk/dnssd/DNSSD.java

```diff
@@ -602,7 +602,7 @@ public static int getIfIndexForName(String ifName) {
         Map<String, String> result = new HashMap<>(record.size());
         for (int i = 0; i < record.size(); i++) {
             try {
-                if (!TextUtils.isEmpty(record.getKey(i)) && !TextUtils.isEmpty(record.getValueAsString(i))) {
+                if (!TextUtils.isEmpty(record.getKey(i)) && TextUtils.isEmpty(record.getValueAsString(i))) {
```

Comment:
Why did you make this change? I can see txt records being passed through on 0.9.14 but not on 0.9.15 because of this change. After this change you will only add the record to the result if the value **is** empty – you should be adding it only if it is **not** empty.

Reply:
My bad 😬 
I want to remove `&& !TextUtils.isEmpty(record.getValueAsString(i))` but some somewhy I removed just `!`
Probably we need test here

Reply:
Hotfix [0.9.16](https://github.com/andriydruk/RxDNSSD/releases/tag/0.9.16) published on Maven Central
Thanks for your help!

Reply:
Great, thanks for the quick turnaround!